packages/lesswrong/components/surveys/SurveyAdminPage.tsx

```diff
@@ -0,0 +1,211 @@
+  surveyList: {
+    display: "flex",
+    flexDirection: "column",
+    gap: "6px",
+    marginBottom: 16,
+  },
+  link: {
+    color: theme.palette.primary.main,
+    fontWeight: 500,
+  },
+  secondaryText: {
+    color: theme.palette.grey[600],
+  },
+  error: {
+    color: theme.palette.text.error2,
+  },
+});
+
+const SurveysEditor = ({classes}: {
+  classes: ClassesType<typeof styles>,
+}) => {
+  const [showCreateSurveyModal, setShowCreateSurveyModal] = useState(false);
+  const [newSurveyName, setNewSurveyName] = useState("");
+
+  const {
+    results: surveys,
+    loading: loadingSurveys,
+    loadMoreProps: loadMoreSurveysProps,
+    refetch: refetchSurveys,
+  } = useMulti({
+    collectionName: "Surveys",
+    fragmentName: "SurveyMinimumInfo",
+    terms: {
+      view: "surveysByCreatedAt",
+    },
+  });
+
+  const {
+    create: createSurvey,
+    loading: loadingCreateSurvey,
+    error: createSurveyError,
+  } = useCreate({
+    collectionName: "Surveys",
+    fragmentName: "SurveyMinimumInfo",
+  });
+
+  const {
+    results: surveySchedules,
+    loading: loadingSurveySchedules,
+    loadMoreProps: loadMoreSurveySchedulesProps,
+  } = useMulti({
+    collectionName: "SurveySchedules",
+    fragmentName: "SurveyScheduleEdit",
+    terms: {
+      view: "surveySchedulesByCreatedAt",
+    },
+  });
+
+  const onOpenCreateSurveyModal = useCallback(() => {
+    setShowCreateSurveyModal(true);
+  }, []);
+
+  const onCloseCreateSurveyModal = useCallback(() => {
+    setShowCreateSurveyModal(false);
+  }, []);
+
+  const onCreateSurvey = useCallback(async () => {
+    await createSurvey({
+      data: {
+        name: newSurveyName,
+      },
+    });
+    await refetchSurveys();
+    setNewSurveyName("");
+    setShowCreateSurveyModal(false);
+  }, [createSurvey, refetchSurveys, newSurveyName]);
+
+  const onNewSurveySubmit = useCallback((e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    if (newSurveyName.length) {
+      void onCreateSurvey();
+    }
+  }, [newSurveyName, onCreateSurvey]);
+
+  const {
+    SingleColumnSection, SectionTitle, Loading, EAButton, FormatDate,
+    BlurredBackgroundModal, EAOnboardingInput, LoadMore,
+  } = Components;
+  return (
+    <SingleColumnSection className={classes.root}>
+
+      <SectionTitle title="Surveys" />
+      {!!surveys && surveys.length > 0 &&
+        <div className={classes.surveyList}>
+          {surveys.map(({_id, name, createdAt}) =>
+            <div key={_id}>
+              <Link
+                to={`/survey/${_id}/edit`}
+                className={classes.link}
+              >
+                {name} (<FormatDate date={createdAt} includeAgo />)
+              </Link>
+            </div>
+          )}
+        </div>
+      }
+      {!loadingSurveys && !!surveys && surveys.length < 1 &&
+        <div className={classes.secondaryText}>No surveys found</div>
+      }
+      <LoadMore {...loadMoreSurveysProps} />
+      {loadingSurveys && <Loading />}
```

Comment:
![Screenshot 2024-06-14 at 11 03 13](https://github.com/ForumMagnum/ForumMagnum/assets/77623106/3a290f5d-841b-4902-b4aa-42cb482e1038)

Could you put the load more on a separate line? (Probably applies to the survey schedules one too)